src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+            <Text>
+              Try&nbsp;
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/user-preferences"
+              >
+                checking your notification preferences
+              </Link>
+              &nbsp;and managing the&nbsp;
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/configure-events"
+              >
+                notification configuration
+              </Link>
+              &nbsp;for your organization.
+            </Text>
+          </StackItem>
+        </Stack>
+      ) : (
+        <>
+          <Stack>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Text>There are currently no notifications for you.</Text>
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/user-preferences"
+              >
+                Check your Notification Preferences
+              </Link>
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/notificationslog"
+              >
+                View the Event log to see all fired events
+              </Link>
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Text>Contact your organization administrator</Text>
+            </StackItem>
+          </Stack>
+        </>
+      )}
+    </EmptyStateBody>
+  </EmptyState>
+);
+
+const DrawerPanelBase = ({
+  isOrgAdmin,
+  getUserPermissions,
+  toggleDrawer,
+}: DrawerPanelProps) => {
+  const { addWsEventListener } = useChrome();
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [isFilterDropdownOpen, setIsFilterDropdownOpen] = useState(false);
+  const [activeFilters, setActiveFilters] = useAtom(
+    notificationDrawerFilterAtom
+  );
+  const navigate = useNavigate();
+  const populateNotifications = useSetAtom(notificationDrawerDataAtom);
+  const addNotification = useSetAtom(addNotificationAtom);
+  const notifications = useAtomValue(notificationDrawerDataAtom);
+  const selectedNotifications = useAtomValue(notificationDrawerSelectedAtom);
+  const updateSelectedNotification = useSetAtom(updateNotificationSelectedAtom);
+  const [hasNotificationsPermissions, setHasNotificationsPermissions] =
+    useState(false);
+  const updateNotificationRead = useSetAtom(updateNotificationReadAtom);
+  const updateAllNotificationsSelected = useSetAtom(
+    updateNotificationsSelectedAtom
+  );
+  const [filterConfig, setFilterConfig] = useState<FilterConfigItem[]>([]);
+  const eventType: ChromeWsEventTypes =
+    'com.redhat.console.notifications.drawer';
+  const handleWsEvent = (event: ChromeWsPayload<NotificationsPayload>) => {
```

Comment:
The event payload should have the `NotificationData` as the generic type. You won't need the zas unknown as NotificationDataz in the `addNotification` call.

```
  const handleWsEvent = (event: ChromeWsPayload<NotificationData>) => {
```